src/components/VaccinationList.jsx

```diff
@@ -0,0 +1,207 @@
+import styled from "styled-components";
+import { useState, useEffect } from "react";
+import GlobalStyles, { Button } from "../GlobalStyles/GlobalStyles";
+import { saveToLocal, loadFromLocal } from "../lib/localStorage";
+import Vaccination from "../Images/Vaccination.jpg";
+
+function VaccinationList() {
+  const initialVaccinations = {
+    hepatitis: {
+      value: "Hepatitis",
+      checked: false,
+      date: "",
+    },
+    leptospirose: {
+      value: "Leptospirose",
+      checked: false,
+      date: "",
+    },
+
+    parainfluenza: {
+      value: "Parainfluenza",
+      checked: false,
+      date: "",
+    },
+
+    pavovirose: {
+      value: "Pavovirose",
+      checked: false,
+      date: "",
+    },
+
+    staupe: {
+      value: "Staupe",
+      checked: false,
+      date: "",
+    },
+
+    tollwut: {
+      value: "Tollwut",
+      checked: false,
+      date: "",
+    },
+  };
+  const localStorageVaccinations = loadFromLocal("_vaccinations");
+  const [vaccination, setVaccination] = useState(
+    localStorageVaccinations ?? initialVaccinations
+  );
+
+  const vaccinationKeys = Object.keys(initialVaccinations);
+  const [showMyVaccinations, setShowMyVaccinations] = useState(false);
+  const [showDate, setShowDate] = useState(false);
+
+  const handleChange = (event) => {
+    let inputValue = event.target.checked;
+    let vaccinationKey = event.target.value.toLowerCase();
+    setVaccination({
+      ...vaccination,
+      [vaccinationKey]: { ...vaccination[vaccinationKey], checked: inputValue },
+    });
+  };
+
+  const handleChangeDate = (vaccinationKey, date) => {
+    setVaccination({
+      ...vaccination,
+      [vaccinationKey]: { ...vaccination[vaccinationKey], date },
+    });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    saveToLocal("_vaccinations", vaccination);
+  };
```

Comment:
Maybe you should close the vaccination card after a 'save' like clicking on close button.

```suggestion
  const handleSubmit = (event) => {
    event.preventDefault();
    saveToLocal("_vaccinations", vaccination);
    handleMyVaccinations()
  };
```